### DISCUSSION TITLE
MEETING: Monthly EMu Users - 15 August 2023

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#880] [MEETINGS]
# Agenda: Monthly EMu User Meeting
## MorphoSource reminders
- [https://github.com/fieldmuseum/EMu-Documentation/discussions/350](https://github.com/fieldmuseum/EMu-Documentation/discussions/350#which-pulication-status-should-fmnh-media-have)
- If contributors need their data to be embargoed:
- A Media Agreement documenting embargo & dates needs to be in EMu
- On MorphoSource, contributor sets embargoed media’s publication status to ‘Private’ (not “Publish with Restricted Download”)

## Multimedia questions in NetX Working group call
- Moving forward if you have questions about emultimedia and media in general we will be handling those in the bi-weekly NetX call. 
- Please post questions here:[NetX Working Group Questions and Answers](https://docs.google.com/document/d/1j339qxi6qXkQ9m1c20WIko9C9t5FgEYy54IRgxYOnFg/edit)

## Transactions Notifications now being sent external to FMNH emails
- Check Transactions notifications.
- If you fill in the “To be notified” fields automated emails will now be sent to external email addresses if you do NOT set an end date on the tab, even if you close the loan on the dates tab.
- - Example of email: From: emu
- - Sent: Monday, August 7, 2023 10:02 AM
- - To: Shepard, William D
- - Subject: EMu Notifications
- - Notification Reports for 07 Aug 2023
- - Dear William,
- - EMu notification reports for 07 Aug 2023.
- - LOAN DUE DETAILS - efmnhtransactions	
- - Processed Date: 07 Sep 2021
- - End Date:       07 Sep 2023
- - IRN:    68568
- - Invoice Number: 53042
- - Due Date:       07 Sep 2023

- What’s your oldest open loan?

## IPT hand holding session
- Date/Time tbd Sept!
- Slap name here
- - Sean Keogh 
- - Rosana Cunha 

### Anthro:  [ED,JK]
- All perfect in the world of Anthro-EMu

### Botany: [WG, KH, TW]
- TW / label Reports, labels.
- Botany needs to regroup to get some consensus/specifications. It doesn’t need to be everyone the same but know who needs what.

#### Paleobotany [MD]
- Imaging scratch - envelopes images mystery solved/cross-checked
- - Most were uploaded and now clearing the rest into EMu.

#### Fossil Verts [CM]
- Chris M. is now working down in IT.

#### Herps: [RK, JM]
- All good in herps

#### Fishes: [SM, KS]
- Ditto all good in fishes

#### IZ: [KGJ, SK]
- Needs to get back into bi-monthly ipt cycle in synch with Insects preferably.

#### Insects:  [RB, MT, NS]
- All hunky dory.
- Nina S. to share a draft of the LinEpig data that will be moved over to narratives.

#### Mammals: [AF,RB]
- Tanzanian Mammals Id key - interns will be finished this week. They have questions for Adam
- Rebecca has shared the images with us for upload to EMu.

### Photo Archives: [RW,NC]
- Getting close to finishing the JW “stuff
- SUE femur coring photos are in EMu.
- Will need to be linked in ecat and econs.
- Filemaker!!!!!  [regrouping later this week, mapping some mm + usage-related data to EMu]

### Exhibitions: [MG,CM,SD]
- Reconciling PS Fossil Verts with Exhibitions.
- NetX - exobj measurements are in dams-test; going into live this week
- - E.g. - https://dams-test.fieldmuseum.tech/app/#asset/144244 
- Registrars are doing import training
- Who should be long term exhibitions contacts, for example after shows close and/or the developer leaves?
- - Contact Exhibitions Registration power user and/or ExhibitionsDev EMu Power user.
- Interns have made a 1st pass in the galleries for mammals specimens.
- Requesting an Exhibitions specific NetX training.

### IT 
- Cross check and cleanup Imaging_Scratch.

#### NetX
[NetX training](https://docs.google.com/spreadsheets/d/1ExCyeugFWHZypJhGo2aP42d4rPAaa4OYj0kKVheWh8o/edit#gid=1505273758&range=A1) 

#### [EXISTING cleanup TASKS](https://github.com/fieldmuseum/EMu-Documentation/labels/data%20cleanup)
- Check the outstanding things please!

##### ipt - Redacting Sensitive data
- TASK: Provide information 
- - Inverts and Herps - Yay, thanks.
- - Geology - Thanks, but can you drop your comments into github please.
- - Everyone else???? 
##### esites
- Cniezgoda has shared a cleaned and standardardized set of site records for Bolivia Departments. Please use those moving forward.
- - TASK: [Cleanup existing Bolivia records that have a rank of “pd2”](https://github.com/fieldmuseum/EMu-Documentation/issues/881)

**Comment by rondlg** [comment]
- [Question 87: Adding collectors to an Accession [KS] github #883](https://github.com/fieldmuseum/EMu-Documentation/discussions/883)
- [Question 88: If I deaccession a specimen but I want to keep it in the database because it was part of a project, what should I be doing? [KMH botany] github #884](https://github.com/fieldmuseum/EMu-Documentation/discussions/884)

**Comment by fmjjones** [comment]
TASK: Please review your records and decide if the notification is required.

Here's a summary of the catalogues with [People to be notified] filled in on the Dates Tab of the Transaction (transaction type) records:
Summary Report - "Catalog"

1	16	Anthropology
2	1	Birds
3	1	Botany
4	90	Exhibitions
5	16	Fossil Invertebrates
6	1	Fossil Vertebrates
7	3688	Insects
8	64	Invertebrate Zoology
9	58	Mammals
10	12	Paleobotany
11	2	Physical Geology
	3949	Total

The conditions that must be met in order for a notification email to be sent (to anyone)
There must be a Parties record linked in the the People to be notified field
![image](https://github.com/fieldmuseum/EMu-Documentation/assets/708594/49de40c5-f865-48dc-870e-c3377545e29b)
That record must have a valid email address in the Email field
![image](https://github.com/fieldmuseum/EMu-Documentation/assets/708594/bd5ae2e4-cec8-46ce-ac71-d5c98d4abca7)
And the loan must be open, the Loan Closed Date must be blank and/or the Cancellation Date must be blank
![image](https://github.com/fieldmuseum/EMu-Documentation/assets/708594/74ddf105-8339-4600-a141-081671240452)

**Reply by rondlg** [reply]
data cleanup task created: https://github.com/fieldmuseum/EMu-Documentation/issues/885
